spec/validators/claim/base_claim_validator_spec.rb

```diff
@@ -224,10 +224,15 @@
     end
 
     it 'should error if invalid' do
-      claim.case_number = 'T87654321'
+      claim.case_number = 'T87654321?'
       should_error_with(claim, :case_number, 'invalid')
     end
 
+    it 'should error if case number has the incorrect format' do
```

Comment:
given the cyclomatic complexity of the case number validation now I would look to extend this/these spec examples to more extensively test the regex

e.g.

```
context 'with unique reference numbers' do
  # test boundaries
  it 'invalid when less than 9 chars' do
  ...
 end
  it 'invalid when more than 20 chars' do
  ...
 end
  it 'invalid when includes symbol' do
  ...
 end
end

context 'with case number' do
...
end
```

Reply:
👍